packages/models-library/src/models_library/service_settings.py

```diff
@@ -59,11 +66,145 @@ class Config:
         }
 
 
-class ServiceSettings(BaseModel):
-    __root__: List[ServiceSetting]
+class SimcoreServiceSettings(BaseModel):
+    __root__: List[SimcoreServiceSetting]
```

Comment:
MINOR: do you think it is practical to have *such a long names* and only an ``s`` difference between them? Since the name is already so long,  I think it would help to call it something like ``SimcoreServiceSettingsList``?

Reply:
Most entries in this module got renamed. Please check if you like it better now.